packages/axelar-std/Cargo.toml

```
@@ -19,6 +19,7 @@ stellar-axelar-std-derive = { workspace = true, optional = true }
 goldie = { workspace = true }
 paste = { workspace = true }
 soroban-sdk = { workspace = true, features = ["testutils"] }
+stellar-axelar-std = { path = "../axelar-std", features = ["testutils"] }
```

```suggestion
stellar-axelar-std = { workspace = true, features = ["testutils"] }
```
refer to the self dependency pattern elsewhere